june/16/karankartikeya.md

```diff
@@ -0,0 +1,12 @@
+# Yesterday
+- I learned about various online editors like Glitch and Codepen
+- I commented on issue #27 in the nostalgify for my preview of the Navbar
+- I learned about the sticky navbar and fixed navbar
+
+# Today
+- I issued a PR to the User interface for adding the navbar page
+- I will try to read the Backend scripts which have been merged
+
+# Blockers
+- Is it good to start reading the Backend scripts for learning `Backend` ?
+    -If not, then guide me how to start with it
```

Comment:
Reply to all the blockers would be given after everyone has submitted their introspect.